### ISSUE TITLE
Remaining time per deck

### ISSUE THREAD

**Issue by khonkhortisan** [Arthur-Milchior/anki-enhance-main-window#118]
![deckETAconcept2](https://user-images.githubusercontent.com/1996449/194724528-361d9049-668e-4733-aeaa-db1730bf017e.png)
With four columns, a 500-card deck of reviews, 100 review limit, a half hour per 100 cards, and starting at noon, it would be
- remaining time for today's cards - 30m
- ETA for today's cards - 12:30 ![imagen](https://user-images.githubusercontent.com/1996449/194727009-dfb8e021-291c-4125-88ab-8e9f12846a82.png)
- remaining time for all unsuspended cards - 2.5h
- ETA for all unsuspended cards - 14:30 ![imagen](https://user-images.githubusercontent.com/1996449/194727015-ebedc052-f14c-49c1-956a-419ee3eefb6b.png)

[Enhance main window](https://ankiweb.net/shared/info/877182321), [Remaining time (for Anki 2.1)](https://ankiweb.net/shared/info/1508357010), [Progress Bar, cards done, cards left, seconds/card, time spent/left, and ETA](https://ankiweb.net/shared/info/1097423555), [Life Drain](https://ankiweb.net/shared/info/715575551), [sectograph](https://sectograph.com/) [napchart](https://napchart.com) [canva](https://www.canva.com/graphs/pie-charts/)

**Comment by siriak** [comment]
Shouldn't ETA for all unsuspended cards be 5 days from now? Because it's when you will finish studying new cards due to 100 review limit

**Comment by Arthur-Milchior** [comment]
I'm sorry @khonkhortisan , but I don't understand your request. Is it a bug report that the time is wrong ? Is it a request for a new feature, and examples of other add-ons implementing it.
To be honest, I'm amazed this add-on still works. I had no time for it in years, literally.

**Comment by khonkhortisan** [comment]
Yes I did the math wrong. This is somewhere between a feature concept and a feature request. Progress Bar and Remaining Time both deal with ETAs, Enhance main window deals only with counts. I thought it would be cool to combine them. Whether ETA means just today's cards or to complete the subdeck, those could be separate columns.
